bears/general/TextLintBear.py

```diff
@@ -0,0 +1,238 @@
+import json
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.settings.Setting import typed_list
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+from dependency_management.requirements.PipRequirement import PipRequirement
+
+
+@linter(executable='textlint',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+):(?P<column>\d+)(?:\s|\u2713)*'
+                     r'(?P<severity>error|warning)\s+(?P<message>.+?)'
+                     r'(?:  .*|\n|$)')
+class TextLintBear:
+    """
+    The pluggable linting tool for text and markdown. It is similar to
+    ESLint, but textlint for natural language.
```

Comment:
"but textlint for natural language"? I mean you are just describing `TextLint` and referring to itself again? :)
Maybe just: `It is similar to ESLint, but covers natural language (instead).` (The "instead" is optional)